apps/studio/Dockerfile

```diff
@@ -21,17 +21,17 @@ ARG BASE_URL_PLACEHOLDER
 RUN apk add --no-cache libc6-compat
 RUN apk update
 WORKDIR /app
- 
+RUN npm install --global pnpm@latest-10
+
 # First install the dependencies (as they change less often)
 
 COPY .gitignore .gitignore
 COPY --from=builder /app/out/json/ .
-COPY --from=builder /app/out/package-lock.json ./package-lock.json
-RUN PUPPETEER_SKIP_DOWNLOAD=true npm ci
- 
+RUN PUPPETEER_SKIP_DOWNLOAD=true pnpm install
+
 # Build the project
 COPY --from=builder /app/out/full/ .
-RUN PUBLIC_URL=${BASE_URL_PLACEHOLDER} npm run build:studio
+RUN PUBLIC_URL=${BASE_URL_PLACEHOLDER} NEXT_CONFIG_OUTPUT=export pnpm run build:studio
```

Comment:
Use [static HTML export](https://nextjs.org/docs/14/app/building-your-application/deploying#static-html-export) so that it works with the nginx-based setup.

This [has some limitations](https://nextjs.org/docs/14/app/building-your-application/deploying/static-exports#unsupported-features), but none of them seem to apply here at the moment.

A potential future improvement would be to re-engineer the docker setup, but I'd like to get it working again first.